narayana-release-process.sh

```diff
@@ -139,6 +139,11 @@ fi
 
 rm -rf $PWD/localm2repo
 ./build.sh clean install -Dmaven.repo.local=${PWD}/localm2repo -Prelease
+if [[ $? != 0 ]]
+then
```

Comment:
Sorry @tomjenkinson , I missed this PR. I will include it in a new PR

Reply:
Thanks @marcosgopen - I think you are doing something like this already now from your pull request https://github.com/jbosstm/narayana/pull/2266/files#diff-530cdb1e8542754f0d5c3b24ac1598f52957ed33177c8ca69b719666077c3bbeR88. As such, I think we can close this pull request but please let me know if you disagree that we are doing this type of check already otherwise feel free to close.

Reply:
Oh, yes, you are right. Thanks for rising this PR.